styles/_mixins.scss

```diff
@@ -12,4 +12,8 @@
     top: 50%;
     transform: translate(0, -50%);
   }
+}
+
+@mixin chatOpeningTransition($property) {
+  transition: $property 0.5s ease;
```

Comment:
Yeah, the idea is that whatever needs to animate in response to the chat opening animation should use this mixin to ensure the duration and easing are the same. Otherwise the animations are not in sync. As of now, this is being used by Chat and then the page container and the modals. It's very possible some other areas will pop up that need to use it as well.